### PR TITLE
2.2 release documentation.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: docs/release_notes/release_notes.md [hunk 1]
```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
-## Deprecation Warnings
+### Incremental Updates
+
+In Katello 2.1 we provided a first look at incremental updates via the API. With the 2.2 release, incremental updates can be applied via the API, UI and CLI. Incremental updates are applying errata, packages or puppet modules to a content view which produces a minor version of that content view version. The updated version can then be pushed out to all the environments that that major version exists in, thereby making small updates available to any hosts that are using that particular major version of the content view. Users can find this functionality by visiting the errata details page and clicking on the 'Content Hosts' tab.
```
Comment (daviddavis) [inline docs/release_notes/release_notes.md]: Easier to read:

s/that that major/that the major/

File: docs/release_notes/release_notes.md [hunk 2]
```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
-## Deprecation Warnings
+### Incremental Updates
+
+In Katello 2.1 we provided a first look at incremental updates via the API. With the 2.2 release, incremental updates can be applied via the API, UI and CLI. Incremental updates are applying errata, packages or puppet modules to a content view which produces a minor version of that content view version. The updated version can then be pushed out to all the environments that that major version exists in, thereby making small updates available to any hosts that are using that particular major version of the content view. Users can find this functionality by visiting the errata details page and clicking on the 'Content Hosts' tab. 
+
+See [Errata User Guide](http://www.katello.org/docs/2.2/user_guide/errata/index.html#applying-errata) for more information.
```
Comment (daviddavis) [inline docs/release_notes/release_notes.md]: See the Errata User Guide

File: docs/release_notes/release_notes.md [hunk 3]
```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
...
+
+### Isolated Capsules
+
+The end goal is for all client communication to be able to go through a Capsule server. Isolating communication through the Capsule prevents the client from needing network connectivity to the Katello server. With a fully configured Capsule, Hosts would boot and get their IP information from the Capsule’s DHCP server, PXE boot from the Capsule’s tftp server, resolve hostnames using the Capsule’s DNS server, provision from kickstart trees that have been synced to the Capsule, configure itself using the Puppet Master running on the Capsule, and recieve content updates directly from the Capsule itself.
```
Comment (daviddavis) [inline docs/release_notes/release_notes.md]: s/recieve/receive/

File: docs/release_notes/release_notes.md [hunk 4]
```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
...
+
+See [Capsules User Guide](http://www.katello.org/docs/2.2/user_guide/capsules/index.html) for more information.
+
+### Docker
+
+Katello 2.2 adds more Docker functionality to the initial set of features provided in 2.1. With 2.2, users can now retreive Docker content from a variety of sources such as Docker hub, private Docker registries, the Red Hat CDN, etc. Docker content can then be published and promoted via Content Views and then pulled or proivisioned to a server running Docker. Docker content can then be inspected at the repository, environment and content view level by viewing the details of each.
```
Comment (daviddavis) [inline docs/release_notes/release_notes.md]: You might want to mention the Docker Tags page too.

**Comment by daviddavis** [comment]
ACK pending comments

File: docs/release_notes/release_notes.md (hunk 1 above)
Comment (waldenraines) [inline docs/release_notes/release_notes.md]: We can currently only apply errata through the UI, not sure if you want to mention that or just mention errata for now.

>  by visiting the errata details page

Maybe say by visiting the errata index page?  I wasn't sure what errata details page you meant, my first thought was from within the Content Host pages for some reason.

File: docs/release_notes/release_notes.md [hunk 5]
```diff
@@ -14,7 +14,43 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
-## Deprecation Warnings
+### Incremental Updates
+
+In Katello 2.1 we provided a first look at incremental updates via the API. With the 2.2 release, incremental updates can be applied via the API, UI and CLI. Incremental updates are applying errata, packages or puppet modules to a content view which produces a minor version of that content view version. The updated version can then be pushed out to all the environments that the major version exists in, thereby making small updates available to any hosts that are using that particular major version of the content view. Users can find this functionality by visiting the errata index page and using either the bulk actions or viewing the details of an individual errata and clicking on the 'Content Hosts' tab.
```
Comment (waldenraines) [inline docs/release_notes/release_notes.md]: The UI doesn't support packages or puppet modules yet and this makes it sound like it does.  Perhaps we just remove "packages or puppet modules" for now?

**Comment by waldenraines** [comment]
ACK from me as well.

**Pull request merged**